tests/views/test_message.py

```diff
@@ -758,14 +754,11 @@ def test_conversation_fail(self, mock_request, mock_get_jwt):
     @patch("response_operations_ui.controllers.message_controllers._get_jwt")
     @patch("flask_login.utils._get_user")
     def test_conversation_reply(self, mock_request, current_user, mock_get_jwt):
-        with self.client.session_transaction() as session:
-            session["messages_survey_selection"] = "ASHE"
+        self._client_session_("Ashe")
```

Comment:
Is this not meant to be caps?

Reply:
I was just following the other tests but yeah you are correct. I've updated them